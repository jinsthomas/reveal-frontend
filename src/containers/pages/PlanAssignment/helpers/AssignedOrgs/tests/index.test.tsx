import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import { AssignedOrgs } from '../index';

describe('PlanAssignment/AssignedOrgs', () => {
  beforeEach(() => {
    jest.resetAllMocks();
  });

  it('renders when no orgs', () => {
    const wrapper = mount(<AssignedOrgs displayLimit={2} id="42" orgs={[]} />);
    expect(toJson(wrapper.find('span'))).toMatchSnapshot('Org names');
    expect(toJson(wrapper.find('Tooltip'))).toMatchSnapshot('Tooltip');
    wrapper.unmount();
  });
});
