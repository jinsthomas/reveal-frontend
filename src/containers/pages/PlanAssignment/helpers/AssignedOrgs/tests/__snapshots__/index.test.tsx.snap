// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlanAssignment/AssignedOrgs renders when no orgs: Org names 1`] = `null`;

exports[`PlanAssignment/AssignedOrgs renders when no orgs: Tooltip 1`] = `null`;
